src/mscorlib/shared/System/Reflection/MemberInfoSerializationHolder.cs

```diff
@@ -9,7 +9,6 @@
 
 namespace System.Reflection
 {
-    [Serializable]
 #if CORECLR
```

Comment:
I'll include that with general ISerializable cleanup (I've updated dotnet/corefx#19119 to include that)